Platform/ArrowlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -801,6 +802,11 @@ UpdateFspConfig (
       FspsConfig->PcieGen4EqPh1UpTxPreset[2] = 0x7;
       FspsConfig->PcieGen4EqPh1UpTxPreset[4] = 0x7;
       FspsConfig->Usb4CmMode                 = 0x0;
+      FspsConfig->PcieRpLtrEnable[0] = 0x1;
+      FspsConfig->PcieRpLtrEnable[2] = 0x1;
+      FspsConfig->PcieRpLtrEnable[3] = 0x1;
+      FspsConfig->PcieRpLtrEnable[4] = 0x1;
+      FspsConfig->PcieRpLtrEnable[5] = 0x1;
```

Comment:
ARL added config data for PcieRpLtrEnable, we should update the config data instead of hard-coded these values.

Reply:
Will update the values in config data instead